### DISCUSSION TITLE
Unable query private tags using RESTFUL API on DCM4CHEE-ARC

### DISCUSSION THREAD

**Discussion by oliverfchen** [dcm4che/dcm4chee-arc-light#3001] [Q&A] [answered]
I have DCM4CHEE ARC light hold Private tags in instances. 
Private Creator (7777,0010)
Patient Create DateTime (7777,1010)
Patient Update Date Time (7777,1011)
I tried using RESTFUL API /aets/DCM4CHEE/rs/series and PYNETDICOM c-find without success. How can I query these values without download DCM files?

**Comment by gunterze** [comment, marked as answer]
dcm4chee proprietary private query attributes are only returned by dicomWeb QIDO with query parameter `includefield=all`, not by DUL C-FIND service.
